UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Helpers/FileSystemHelper.cs

```diff
@@ -108,14 +109,13 @@ public bool DownloadReadmeFile(string filePath, Stream stream, string lineToWrit
         {
             if (stream != null)
             {
+                var extendedAsciiEncoding = Encoding.GetEncoding("iso-8859-1");
                 CreateFileCopy(filePath, stream);
-                using StreamReader reader = new StreamReader(stream);
-                string secondLineText = string.Empty;
-                secondLineText = GetLine(filePath);
-                string text = File.ReadAllText(filePath);
+                var text = File.ReadAllText(filePath, extendedAsciiEncoding);
+                var secondLineText = GetLine(filePath);
```

Comment:
GetLine also reads filePath. I think that this should use extendedAsciiEncoding too, so that the Replace method below finds the match between text and secondLineText